packages/eslint-plugin-angular-template-spacing/rollup.config.js

```diff
@@ -1,21 +1,34 @@
 import json from '@rollup/plugin-json';
 import { createBaseRollupConfig } from '../../rollup.helper';
+import commonjs from "@rollup/plugin-commonjs";
```

Comment:
what is this plugin for?

Reply:
Without it, i keep getting this [error](https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module) when bundling the schematics. Any other way to solve it?